index.html

```diff
@@ -3409,6 +3409,44 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+      <section>
+        <h3>Media Types</h3>
+
+        <p>
+          Media types as defined in [[RFC6838]] serve a useful purpose with verifiable 
+          credentials, especially in regards to identifcation of a particular syntax 
+          that is in use with a verifiable credential.
+        </p>
+        <p>
+          Syntaxes SHOULD be identified by a media type, and certain conventions as 
+          outlined in this section should be followed when defining or using media types 
```

Comment:
Ought this `should` be `SHOULD`, or a different word to avoid confusion?
```suggestion
          outlined in this section should be followed when defining or using media types 
```

Reply:
i think `SHOULD` but that was part of the reason i wanted this as a draft first - happy to adjust it so.

Reply:
I feel this paragraph should speak of "serializations / bytes"... also less might be more on this front.

Reply:
@OR13 agree - i think `serializations` is better but given that the `syntaxes` section is where this is at the moment I followed the current language of the spec.  would you be open to an issue on the topic of terminology related to syntax vs serialization, etc and how to handle that done separately?

Reply:
@TallTed does [this suggestion](https://github.com/w3c/vc-data-model/pull/1055#discussion_r1122204405) work? if so i will commit it.

Reply:
Marking resolved based on commit and thumbs up.  thanks @TallTed